courts_db/data/courts.json

```diff
@@ -29498,7 +29778,8 @@
         "regex": [
             "New York Court of Claims",
             "Court of Claims? of New York",
-            "New York Claims Court"
+            "New York Claims Court",
+            "Court of Claims"
```

Comment:
while this is probably how its stated - im not sure we can do court of claims here - its a bit ... tough because the court of claims is also a federal court.